.github/workflows/changelog.yml

```
@@ -11,28 +11,39 @@ on:
   PR_NUMBER: ${{ github.event.number }}
 
 jobs:
   changelog:
-    runs-on: self-hosted
+    runs-on: ubuntu-latest # Pirate
     if: github.event.pull_request.merged == true
     permissions:
       contents: write
     steps:
       - name: Checkout Master
         uses: actions/checkout@v3
```

actionlint (1.7.11)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)